packages/components/src/Tag/Tag.tsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from "react";
+import Text from "../Text";
+import { TypographyColor } from "../common/typography";
+import cx from "classnames";
+import styles from "./Tag.module.css";
+
+export type Color = Omit<
```

Comment:
I think `Omit` only works on object types / interfaces, I'm seeing a weird type for `Color` after it:
<img width="439" alt="Screen Shot 2021-07-23 at 2 25 50 PM" src="https://user-images.githubusercontent.com/15840841/126825517-81fbd374-2210-4a1c-8161-732ae42a0350.png">

I'm also not finding any documentation around picking out of a union of strings like this ... it may be most straightforward to just explicitly type it here 😬